KsApi/Service.swift

```diff
@@ -790,4 +790,13 @@ public struct Service: ServiceType {
       .perform(mutation: GraphAPI.WatchProjectMutation(input: GraphAPI.WatchProjectInput.from(input)))
       .flatMap(WatchProjectResponseEnvelope.producer(from:))
   }
+
+  public func blockUser(userId: String)
+    -> SignalProducer<EmptyResponseEnvelope, ErrorEnvelope> {
+    return GraphQL.shared.client
+      .perform(mutation: GraphAPI.BlockUserMutation(input: GraphAPI.BlockUserInput(blockUserId: userId)))
```

Comment:
n.B. `GraphAPI.BlockUserMutation` wants us to pass it a `GraphQLID` type, which can also be a `String`. It's not obvious to me if we'll need to base64-encode the value like a proper GraphQL identifier, or if it will be happy taking a plain, string-based user ID like `"12345"`.

Reply:
Let's follow the same "adapter" pattern that we're using elsewhere here. We can create an extension on `GraphAPI.BlockUserInput` for creating the input type. Extracting this allows us to test it more easily. See `GraphAPI.WatchProjectInput.from(_ input:...` for an example